torch/testing/_internal/common_methods_invocations.py

```diff
@@ -602,7 +602,7 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
     else:
         return (input, )
 
-def sample_inputs_addbmm(op_info, device, dtype, requires_grad):
+def sample_inputs_addbmm(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
I think the sample inputs would've to be changed, as per the changes brought by #53014.
eg. https://github.com/pytorch/pytorch/pull/55201#discussion_r608363961

These tests aren't running properly due to https://github.com/pytorch/pytorch/pull/54945#discussion_r607039433.